data/json/items/melee.json

```diff
@@ -2018,6 +2018,40 @@
         "price" : 8000,
         "qualities": [["COOK", 1]]
     },{
+        "id": "halberd",
+        "type": "GENERIC",
+        "symbol": "/",
+        "color": "light_gray",
+        "name": "halberd",
+        "description": "This is a versatile polearm with an axe blade, a spike, and other fun things attached to a long stick.",
+        "price": 50000,
+        "material": ["wood", "steel"],
+        "flags": ["CHOP", "DURABLE_MELEE", "REACH_ATTACK", "NON_STUCK"],
+        "techniques": ["WBLOCK_1", "WIDE"],
+        "weight": 3175,
+        "volume": 15,
+        "bashing": 16,
+        "cutting": 35,
+        "to_hit": 2,
+        "qualities": [["COOK", 1]]
+  },{
+        "id": "halberd_fake",
+        "type": "GENERIC",
+        "symbol": "/",
+        "color": "light_gray",
+        "name": "halberd",
+        "description": "This is a dull, cheaply made replica of a polearm with an axe blade, a spike, and other fun things attached to a long stick.",
+        "price": 5000,
+        "material": ["wood", "aluminum"],
+        "flags": ["BASH", "REACH_ATTACK"],
```

Comment:
There is no "BASH" flag.
Flags like "CHOP" and "SLICE" don't change the damage type, they only affect sticking chance.
Flags "SPEAR" and "STAB" do change the cutting type to piercing one.